### DISCUSSION TITLE
Welcome to hosts Discussions!

### DISCUSSION THREAD

**Discussion by StevenBlack** [StevenBlack/hosts#1480] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rochmind** [comment]
Is there a list that can block email trackers? At the moment I have created a hosted wild card for diversion that I use on asus merlin based router, adapted from here

https://github.com/cfa999/email_trackers_block

**Comment by funilrys** [comment]
Hi and welcome here (if you are new)!

 I'm Nissar and I think my profile describes me the best. 

Around here, I'm slightly supporting Steve @StevenBlack on some issues. But I'm more active when it comes to code change inside the python tools we provide ☺️.

Feel free to @\ping me if you have any questions, ideas, or changes you want/have to make to the tools we provide!

Cheers!
